src/main/java/com/codessquad/qna/user/UserController.java

```diff
@@ -0,0 +1,86 @@
+package com.codessquad.qna.user;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class UserController {
+    private final List<User> users = new ArrayList<>();
+
+    @GetMapping("/user/form")
+    public String goUserForm(Model model) {
+        model.addAttribute("actionUrl", "/user/create");
+        model.addAttribute("buttonName", "회원가입");
+        return "user/form";
+    }
+
+    @PostMapping("/user/create")
+    public String createUser(HttpServletRequest request) {
```

Comment:
```suggestion
    public String createUser(User user) {
```
@wheejuni @ksundong 
1) 요렇게 할 경우에도 `RequestParam` 붙이는 게 좋을까요? 안 붙여도 괜찮을까요? 
2) 왜 post 로 사용자 생성후 redirect로 다른 페이지로 이동할까요? 만약에 '회원가입이 완료되었습니다' 라는 템플릿을 리턴해 준다면?

Reply:
@honux77
해봤는데, POST method로 request할 때, name? id?값이 일치하면 저렇게도 되네요!
2번은 궁금해서 찾아봤는데, 다른 페이지로 redirect를 시켜주지 않으면 뒤로가기를 눌렀을 때, 같은 요청이 전송될 수 있어서? 그렇게 하는 것 같아요!
https://stackoverflow.com/questions/1083357/do-you-always-redirect-after-post-if-yes-how-do-you-manage-it